packages/@ember/object/index.d.ts

```diff
@@ -1 +1,17 @@
 export let action: MethodDecorator;
+
+// export {
+//   aliasMethod,
+//   computed,
+//   defineProperty,
+//   get,
+//   getProperties,
+//   getWithDefault,
+//   notifyPropertyChange,
+//   observer,
+//   set,
+//   setProperties,
+//   trySet,
+// } from '@ember/-internals/metal';
```

Comment:
Why is this here? In general, I think we'd prefer to avoid landing a commented out section like this (basically we should bring it back, delete it, or add a comment explaining why its commented).

Reply:
It's stuff that we want to export but isn't ready yet. I'll remove it and add it back properly in another PR.